### PR TITLE
mk-configure 0.29.1 (new formula)

### PR DESCRIPTION
**Pull request by cheusov**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----
I'm aware that mkc_check_common.sh is installed to bin/ directory without executable flag. This is correct because this is a shell source file included by executables with a help of "." (AKA) "source". As you may know "." and "source" commands search for files in PATH directories.

### CODE REVIEW

**Comment by cheusov** [comment]
Can anybody please have a look at my comments above?

File: Formula/mk-configure.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  depends_on "makedepend" => :build
+  depends_on "makedepend"
+
+  def install
+    ENV["PREFIX"] = prefix.to_s
+    ENV["MANDIR"] = man.to_s
+
+    system "bmake", "all"
+    system "bmake", "install"
+    cp "presentation/presentation.pdf", "#{share}/doc/mk-configure/"
```
Comment (bfontaine) [inline Formula/mk-configure.rb]: Use `(share/"doc/mk-configure").install "presentation/presentation.pdf"` instead.

File: Formula/mk-configure.rb [hunk 2]
```diff
@@ -0,0 +1,23 @@
+class MkConfigure < Formula
+  desc "Lightweight replacement for GNU autotools"
+  homepage "https://github.com/cheusov/mk-configure"
+  url "https://downloads.sourceforge.net/project/mk-configure/mk-configure/mk-configure-0.29.1/mk-configure-0.29.1.tar.gz"
+  sha256 "a675c532f6a857c79dedef2cce4776dda8becfbe7d2126e5f67175aee56c3957"
+
+  depends_on "bmake" => :build
+  depends_on "makedepend" => :build
+  depends_on "makedepend"
```
Comment (bfontaine) [inline Formula/mk-configure.rb]: Keep only one `depends_on "makedepend"` line here.

File: Formula/mk-configure.rb (hunk 1 above)
Comment (cheusov) [inline Formula/mk-configure.rb]: Done

File: Formula/mk-configure.rb (hunk 2 above)
Comment (cheusov) [inline Formula/mk-configure.rb]: Done. See latest commits in my branch

**Comment by fxcoudert** [comment]
__This is a resubmission of a package that was already submitted in #13442, without saying so. Please do not do that, and provide full history and context on PRs.__

---

@cheusov Thank you for your contribution to Homebrew, but unfortunately that package is not established enough at this point to be accepted. Also, self-submissions are generally frowned upon as you can read in [Acceptable Formulae](https://github.com/Homebrew/brew/blob/master/docs/Acceptable-Formulae.md#niche-or-self-submitted-stuff).

For now, hosting this formula in a [personal tap](https://github.com/Homebrew/brew/blob/master/docs/How-to-Create-and-Maintain-a-Tap.md) might be a good alternative for you.

**Pull request closed without merging**